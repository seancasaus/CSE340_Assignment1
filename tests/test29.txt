TbDvC ((F)|((f).(Z)))|((j).(H)) , umQtJ f , y7 ((((s).(f))|((1).(r)))*).((s).(f)) , SgkKj (z)|(((H)|(x))|((D)|(j))) #
" z1rsffzDf1rjHFjHFfjf1r1rff " 
