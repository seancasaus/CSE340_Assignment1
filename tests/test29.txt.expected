SgkKj , "z"
y7 , "1rsf"
umQtJ , "f"
SgkKj , "z"
SgkKj , "D"
umQtJ , "f"
ERROR
